### DISCUSSION TITLE
Typeform-Rowy-Firebase integration data flow

### DISCUSSION THREAD

**Discussion by jairiza** [AntlerVC/firetable#1398] [Support / Q&A] [answered]
I have connected a webhook that receives the responses from a typeform submission, and transfers them to Firebase through Rowy. The problem is that in Firebase the name of each of the fields in a Typeform submission is randomly generated, in the style: 0472b437-bb85-4886-968f-1b265d89087f and I cannot identify to which Typeform question that field corresponds each time a submission is made since it varies according to the submissions and then in Firebase it is stored alphabetically. Does anyone have a solution for this? Can I somehow assign a unique identifier to each of the fields so that with each submission I know which question each answer is referenced to?

**Comment by Chadha93** [comment]
Hi @jairiza,

I recommend you try BuildShip, to make this data flow integration. 
BuildShip - https://buildship.com

Cheers,
Gaurav

**Reply by Chadha93** [reply, marked as answer]
@jairiza Documentation on Typeform webhook for Rowy - https://docs.rowy.io/webhooks/typeform
